client/blocks/plan-storage/bar.jsx

```diff
@@ -58,7 +58,7 @@ class PlanStorageBar extends Component {
 		return (
 			<div className={ classes }>
 				<ProgressBar
-					className="plan-storage__bar"
+					className="plan-storage__storage-bar"
```

Comment:
But the class is there @retrofox, it's just applied to the wrapping `div` see https://github.com/Automattic/wp-calypso/blob/9d7d0bdfcef5152889901ee525efbb069107bde8/client/blocks/plan-storage/bar.jsx#L51